browser/importer/extensions_import_helpers.cc

```diff
@@ -0,0 +1,93 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/importer/extensions_import_helpers.h"
+
+#include <memory>
+#include <unordered_map>
+
+#include "base/files/file_util.h"
+#include "base/logging.h"
+#include "base/task/thread_pool.h"
+#include "brave/common/importer/importer_constants.h"
+#include "components/value_store/value_store.h"
+#include "components/value_store/value_store_factory.h"
+#include "components/value_store/value_store_factory_impl.h"
+#include "extensions/browser/api/storage/value_store_util.h"
+#include "extensions/browser/extension_file_task_runner.h"
+#include "extensions/common/constants.h"
+
+namespace brave {
+namespace {
+using ExtensionStorageMap =
+    std::unordered_map<std::string, std::unique_ptr<value_store::ValueStore>>;
```

Comment:
maybe `base::flat_map` is preferred over `std::unordered_map`?

Reply:
done